Transactions.md

```diff
@@ -62,7 +64,7 @@ Also known as a CET.
   * `txin[0]` outpoint: `txid` of funding transaction and `output_index` 0
   * `txin[0]` sequence: 0xFFFFFFFE
   * `txin[0]` script bytes: 0
-  * `txin[0]` witness: `0 <signature_for_offer_pubkey> <signature_for_accept_pubkey>`
+  * `txin[0]` witness: `0 <signature_for_pubkey1> <signature_for_pubkey2>`
```

Comment:
you're going to need to link up to the previous section where these are defined as someone linked directly here will no longer know what this means